README.Rmd

```diff
@@ -74,49 +80,37 @@ install the latest release of **polars** on Linux (x86_64) one would use:
 ```r
 install.packages(
   "https://github.com/pola-rs/r-polars/releases/latest/download/polars__x86_64-pc-linux-gnu.gz",
-  repos = NULL 
+  repos = NULL
 )
 ```
 
 Similarly for Windows
-([URL](https://github.com/pola-rs/r-polars/releases/latest/download/polars.zip)) 
+([URL](https://github.com/pola-rs/r-polars/releases/latest/download/polars.zip))
 and MacOS (x86_64,
 [URL](https://github.com/pola-rs/r-polars/releases/latest/download/polars__x86_64-apple-darwin20.tgz)).
 Just remember to invoke the `repos = NULL` argument if you are installing these
 binary builds directly from within R.
 
-One exception worth noting is MacOS (arm64), i.e. systems based on the new M1/M2
-"Silicon" chips. To install **polars** on one of these machines, we need to
```

Comment:
Since you removed this paragraph, does it mean that this PR close https://github.com/pola-rs/r-polars/issues/145?

Reply:
Indeed.

Obviously that is not being maintained, so I think it would be better to have a source install.
Also, a successful CRAN release will host the arm64 macOS binary package on CRAN.

Reply:
ok, I updated the first comment of the PR